pipelines/skylab/multiome/Multiome.changelog.md

```diff
@@ -1,3 +1,10 @@
+# 3.0.0
```

Comment:
We should update the pipeline version in the Multiome Overview doc

Reply:
Thank you! updated it